callback-express/package-old.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Did you intend to retain package-old.json?

Reply:
good catch - fixed